huaweicloud/services/acceptance/ces/resource_huaweicloud_ces_alarmrule_test.go

```diff
@@ -308,6 +364,112 @@ resource "huaweicloud_ces_alarmrule" "alarmrule_1" {
     value               = 1
     unit                = "count"
     count               = 1
+    suppress_duration   = 0
+  }
+
+  alarm_actions {
+    type              = "notification"
+    notification_list = [
+      huaweicloud_smn_topic.topic_1.topic_urn
+    ]
+  }
+}
+`, testCESAlarmRule_base(rName), rName)
+}
+
+func testCESAlarmRule_multiConditions(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_ces_alarmrule" "alarmrule_1" {
+  alarm_name           = "rule-%s"
+  alarm_action_enabled = true
+  alarm_type           = "MULTI_INSTANCE"
+
+  metric {
+    namespace   = "SYS.ECS"
+
+    dimensions {
+      name  = "instance_id"
+      value = huaweicloud_compute_instance.vm_1.id
+    }
+  }
+
+  condition  {
+    period              = 300
+    filter              = "average"
+    comparison_operator = ">"
+    value               = 6.5
+    unit                = "B/s"
+    count               = 1
+    suppress_duration   = 300
+    metric_name         = "network_incoming_bytes_rate_inband"
+    alarm_level         = 3
+  }
+
+  condition  {
+    period              = 300
+    filter              = "average"
+    comparison_operator = ">"
+    value               = 6.5
+    unit                = "B/s"
+    count               = 1
+    suppress_duration   = 300
+    metric_name         = "network_outgoing_bytes_rate_inband"
```

Comment:
it's better to update the metric_name because we rarely write two conditions that are different in `alarm_level`

Reply:
test case updated